apps/block/templates/_sidebar.jade

```diff
@@ -6,7 +6,7 @@
       include info/_description
       include info/_connections
 
-      if user && user.has('name')
+      if user && user.has('username')
```

Comment:
@broskoski I noticed this check for has username in a few places. Is it actually possible for there to be a user being rendered but them to not have a username?